src/v0/destinations/twitter_ads/config.js

```diff
@@ -0,0 +1,21 @@
+const { getMappingConfig } = require('../../util');
+
+const BASE_URL = 'https://ads-api.twitter.com/12/measurement/conversions';
+
+const ConfigCategories = {
+  TRACK: {
+    type: 'track',
+    name: 'TwitterAdsTrackConfig',
+  },
+};
+
+const mappingConfig = getMappingConfig(ConfigCategories, __dirname);
+
+const authHeaderConstant = "OAuth oauth_consumer_key=\"qwe\", oauth_nonce=\"V1kMh028kZLLhfeYozuL0B45Pcx6LvuW\", oauth_signature=\"Di4cuoGv4PnCMMEeqfWTcqhvdwc%3D\", oauth_signature_method=\"HMAC-SHA1\", oauth_timestamp=\"1685603652\", oauth_token=\"yrdghfvhjvhj\", oauth_version=\"1.0\"";
```

Comment:
Why are we mentioning these values as constant ?

Reply:
If this header is not present, would the event fail ?

Reply:
for test case pupose only.

Reply:
We shouldn't include a constant in source-code that is only to be used in tests. We can probably put it somewhere else

Reply:
As discussed, let's use this constant in tests itself. Resolving this comment for now.